activation_logger.go

```diff
@@ -76,13 +77,19 @@ func NewActivation(config ActivationLoggerConfig) (Logger, error) {
 
 	l := &activationLogger{
 		underlying: config.Underlying,
-
-		activations: config.Activations,
 	}
 
 	return l, nil
 }
 
+func (l *activationLogger) Debugf(ctx context.Context, format string, params ...interface{}) {
+	l.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf(format, params...))
+}
+
```

Comment:
Should we handle nil error? Maybe there are situations when you want to print error message but there are no error with it so maybe `Errorf(ctx, nil, “should work”)`.

Reply:
I'll drop the `stack` part if it's `nil`. :+1: